scripts/variantstore/wdl/GvsUtils.wdl

```diff
@@ -1307,3 +1309,47 @@ task PopulateFilterSetInfo {
     File monitoring_log = "monitoring.log"
   }
 }
+
+# TODO - should put in a do not overwrite that file logic
```

Comment:
is this going to be covered by this PR/ticket or a subsequent one?

Reply:
I'll make a ticket for that.

Reply:
Actually, I ended up doing that work in this PR>